File: app/src/main/java/github/fushaolei/CreateBody.java
package github.fushaolei;

/**
 * @Auther: fushaolei
 * @datetime: 2021/2/11
 * @desc:
 */
public class CreateBody {
    private String message;
    private String content;

    public CreateBody(String message, String content) {
        this.message = message;
        this.content = content;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    @Override
    public String toString() {
        return "CreateBody{" +
                "message='" + message + '\'' +
                ", content='" + content + '\'' +
                '}';
    }
}

File: app/src/main/java/github/fushaolei/entity/CreateBody.kt
package github.fushaolei.entity

/**
 * @Auther: fushaolei
 * @datetime: 2021/2/11
 * @desc:
 */
class CreateBody(var message: String, var content: String) {
    override fun toString(): String {
        return "CreateBody{" +
                "message='" + message + '\'' +
                ", content='" + content + '\'' +
                '}'
    }
}